CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.13] - 2019-01-17
+## [0.13] - 2019-02-13
```

Comment:
0.13 has already been released, please create a new [Unreleased] section for new changes.